streamalert_cli/_infrastructure/main.tf.json

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
please remove this and update these lines to ignore these files:

https://github.com/airbnb/streamalert/blob/7d198a3273781f66465420e90886a3ce53ec7559/.gitignore#L8-L12